tools/scripts/lock-api-state.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash -ex
+
+## lock in current protobuf state
```

Comment:
Can you delete this comment and put some brief documentation (why/when to run this and what it does) in the `tools/scripts/README.md`?

Reply:
I'll be adding docs to the release document but readme sounds good too. I think it'd be good to have some description of the script inline with it.